src/commandlineflags.h

```diff
@@ -10,22 +10,51 @@
 // Macros for declaring flags.
 #define DECLARE_bool(name) extern bool FLAG(name)
 #define DECLARE_int32(name) extern int32_t FLAG(name)
-#define DECLARE_int64(name) extern int64_t FLAG(name)
```

Comment:
Because it's not used?

Reply:
Yes, I can keep it around if you prefer. We are lacking `ParseInt64Flag`/`Int64FromEnv` though.

Reply:
No, i think that okay.